### PR TITLE
Keyboard shortcuts

### PR DESCRIPTION
**Pull request by milesj76**

### CODE REVIEW

**Comment by milesj76** [comment]
Pull request ready for review!

**Pull request info:** https://github.com/felixakiragreen/artsee/issues/10#issuecomment-997508428

File: src/newtab/Frame.svelte [hunk 1]
```diff
@@ -112,6 +113,34 @@
...
+        break
+      case 'LAST':
+        onLast()
+        break
+      default:
+        console.error('Keyboard shortcut dispatch error. Unhandled error.')
+        break
+    }
+  }
+
```
Comment (felixakiragreen) [inline src/newtab/Frame.svelte:143]: Love this! Makes it easy to update later.

File: src/newtab/Frame.svelte [hunk 2]
```diff
@@ -112,6 +113,34 @@
...
+        onRandom()
+        break
+      case 'FIRST':
+        onFirst()
+        break
+      case 'LAST':
+        onLast()
+        break
+      default:
+        console.error('Keyboard shortcut dispatch error. Unhandled error.')
```
Comment (felixakiragreen) [inline src/newtab/Frame.svelte:139]: Small improvement:
```suggestion
        console.error('Keyboard shortcut dispatch error. Unhandled:', action)
```
Then you can see which specifically is unhandled.
Though, I just noticed the log above, so you'd already know. XD

**Review by felixakiragreen — APPROVED** [review]
Looks great!

**Pull request merged**